packages/pwa-devdocs/src/venia-pwa-concept/setup/index.md

```diff
@@ -90,95 +137,82 @@ If you clone the PWA Studio project repo into the `magento2ce` directory of the
 
 ``` sh
 npm install
 ```
 
-## Step 3. Link and install module
+## Step 4. Set environment variables
 
-Navigate to your Magento installation's `app/code/Magento` directory and create a `Pwa` symlink folder linking to the project's `module` directory.
+Under the `packages/venia-concept` directory, copy `.env.dist` into a new `.env` file:
 
-**Example command:**
+**Example commands:**
 ``` sh
-ln -s /Users/magedev/pwa-studio/packages/pwa-module Pwa
+cd /Users/magedev/pwa-studio/packages/venia-concept
 ```
-Or from your Magento 2 root
 ``` sh
-ln -s pwa-studio/packages/pwa-module app/code/Magento/Pwa
+cp .env.dist .env
 ```
 
-### Enable and install
-
-Navigate to your Magento installation's root director and run the following command to enable the module:
+In the `.env` file set the value of `MAGENTO_BACKEND_DOMAIN` to the URL of your Magento development store.
 
-``` sh
-bin/magento module:enable Magento_Pwa
+**Example:**
+``` text
+MAGENTO_BACKEND_DOMAIN="https://magento.test/"
 ```
 
-Install the module using the following command:
-``` sh
-bin/magento setup:upgrade
-```
+## Step 5. Start the server
 
-## Step 4. Link theme directory
+Use any of the following commands from the **project root directory** to start the server:
 
-Navigate to your Magento installation's `app/design/frontend/Magento` directory and create a `venia` symlink folder linking to the project's `theme-frontend-venia` directory.
+`npm run watch:venia`
 
-**Example command:**
-``` sh
-ln -s /Users/magedev/pwa-studio/packages/venia-concept venia
-```
+: Starts the Venia storefront development environment.
 
-## Step 5. Activate the Venia theme
+`npm run watch:all`
 
-Browse to the Admin section of your Magento store and configure it to use the **Magento Venia** theme.
-You can find this configuration using the **Configuration** link in the **Content** tab.
+: Runs the full PWA Studio developer experience, which include Venia hot-reloading and concurrent Buildpack/Peregrine rebuilds.
 
-## Step 6. Set environment variables
+`npm run build && npm run stage:venia`
 
-Under the Venia project's `theme-frontend-venia` directory, copy `.env.dist` into a new `.env` file and update the variables with the URL to your Magento development store.
+: Generates build artifacts and runs the staging environment, which uses more compressed assets and more closely reflects production.
 
-**Example commands:**
-``` sh
-cd /Users/magedev/pwa-studio/packages/venia-concept
-```
-``` sh
-cp .env.dist .env
-```
-
-## Step 7. Start the development server
+### Browsing to the application
 
-Use the following command to start the development server:
+After the development server is up and running, look for a similar line in the terminal output (the port may differ for your instance):
 
 ``` sh
-npm start
+PWADevServer ready at https://magento-venia.local.pwadev:8001
 ```
 
-{: .bs-callout .bs-callout-info}
-**Note:**
-Some users have reported using `sudo npm start` to get around permissions.
```

Comment:
We have to add the *this is strongly discouraged and not supported*. Running `npm` as root risks changing ownership of so many of your Node.js files that basic operations now require sudo permission, and web servers not running as root are unable to access their dependencies. If you find you must run npm with sudo, please report the issue.